backend/src/services/interview/interview.service.ts

```diff
@@ -143,18 +145,29 @@ class Interview {
 
   public async update(data: {
     id: number;
+    user: UserWithPermissions;
     interviewUpdateInfoRequestDto: InterviewsUpdateRequestDto;
   }): Promise<InterviewsByIdResponseDto> {
-    const { id, interviewUpdateInfoRequestDto } = data;
-    const { interviewerUserId, status, interviewDate } =
-      interviewUpdateInfoRequestDto;
+    const { id, interviewUpdateInfoRequestDto, user } = data;
 
-    const interview = await this.#interviewRepository.update({
-      id,
-      interviewerUserId,
-      status,
-      interviewDate,
+    const hasInterviewsPermission = checkHasPermission({
+      permissionKeys: [PermissionKey.MANAGE_INTERVIEWS],
+      userPermissions: user.permissions,
     });
+    const hasInterviewerUserId =
+      'interviewerUserId' in interviewUpdateInfoRequestDto;
```

Comment:
Maybe it would be better to use `.hasOwnProperty` to check if object has a specific prop